source/braille.py

```diff
@@ -327,6 +328,12 @@
 #: Unicode braille indicator at the end of untranslated braille input.
 INPUT_END_IND = " ⣹"
 
+# Delimiters for the start and end of format tags.
+FORMAT_TAG_START_IND = "⣋"
+"""Unicode braille indicator at the start of a formatt tag."""
+FORMAT_TAG_END_IND = "⣙"
+"""Unicode braille indicator at the end of a formatt tag."""
```

Comment:
can you make this a StrEnum?

`class FormatTag(StrEnum):  ...`

Reply:
I just went with the convention established earlier in the file, but am happy to convert it.

Reply:
yep - this is just legacy code before enums were introduced to python. We now  try and group constants into enums where appropriate

Reply:
Makes sense